packages/quantic/cypress/integration/breadcrumb-manager/breadcrumb-manager.cypress.ts

```diff
@@ -0,0 +1,205 @@
+
+        Actions.numerciFacet.checkFirstValue();
+
+        Expect.displayClearFilters(true);
+        Expect.numerciFacet.displayBreadcrumb(true);
+        Expect.numerciFacet.displayLabel(true);
+        Expect.numerciFacet.labelContains('Youtube Likes');
+        Expect.numerciFacet.displayValues(true);
+        Expect.numerciFacet.numberOfValues(1);
+
+        scope('when selecting more values', () => {
+          for (let index = 1; index <= 4; index++) {
+            Actions.numerciFacet.checkValueAt(index);
+
+            Expect.numerciFacet.displayValues(true);
+            Expect.numerciFacet.numberOfValues(index + 1);
+            Expect.numerciFacet.displayShowMore(false);
+          }
+
+          Actions.numerciFacet.checkValueAt(5);
+
+          Expect.numerciFacet.displayValues(true);
+          Expect.numerciFacet.numberOfValues(5);
+          Expect.numerciFacet.displayShowMore(true);
+
+          Actions.clickShowMoreNumericFacetBreadcrumb();
+
+          Expect.numerciFacet.numberOfValues(6);
+          Expect.numerciFacet.displayShowMore(false);
+        });
+
+        scope('when clearing a value', () => {
+          Actions.clickFirstValueNumericFacetBreadcrumb();
+          cy.wait(InterceptAliases.Search);
+
+          Expect.numerciFacet.numberOfValues(5);
+        });
+      });
+
+      scope('when selecting values in time frame facet', () => {
+        Expect.dateFacet.displayBreadcrumb(false);
+
+        Actions.timeFrameFacet.selectValue('Past 6 months');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.dateFacet.displayBreadcrumb(true);
+        Expect.dateFacet.displayLabel(true);
+        Expect.dateFacet.labelContains('Date');
+        Expect.dateFacet.numberOfValues(1);
+        Expect.dateFacet.breadcrumbValueLabelAtIndexContains('Past 6 months');
+
+        Actions.timeFrameFacet.selectValue('Past year');
+        cy.wait(InterceptAliases.Search);
+
+        Expect.dateFacet.displayBreadcrumb(true);
+        Expect.dateFacet.displayLabel(true);
+        Expect.dateFacet.numberOfValues(1);
+        Expect.dateFacet.breadcrumbValueLabelAtIndexContains('Past year');
+      });
+
+      scope('when clearing all filters', () => {
+        Actions.clickClearFilters();
+
+        Expect.numerciFacet.displayBreadcrumb(false);
+        Expect.dateFacet.displayBreadcrumb(false);
+      });
+
+      scope('when selecting values in category facet', () => {
+        Expect.categoryFacet.displayBreadcrumb(false);
+
+        Actions.categoryFacet.selectChildValue('North America');
+
+        Expect.categoryFacet.displayBreadcrumb(true);
+        Expect.categoryFacet.displayLabel(true);
+        Expect.categoryFacet.labelContains('Country');
+        Expect.categoryFacet.displayValues(true);
+        Expect.categoryFacet.numberOfValues(1);
+        Expect.categoryFacet.breadcrumbValueLabelAtIndexContains(
+          'North America'
+        );
+
+        Actions.categoryFacet.selectChildValue('Canada');
```

Comment:
when we're performing more actions then validating the result, we should use more `scope` statements to improve the readability of the report. Otherwise it just looks like we're validating the same thing twice in a row.

Reply:
Oh yes that sounds like a good idea!